.github/workflows/update-seqr-helm-version.yml

```diff
@@ -0,0 +1,39 @@
+name: Update seqr Version
+on:
+  push:
+    branches:
+      - seqr-helm-version
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: get seqr version
+        run: |
+          seqr_version=$(grep "SEQR_VERSION[ ]*=[ ]*'.*'" ./settings.py | sed -n "s/^.*'\(.*\)'/\1/p")
+          commit_id=$(git rev-parse --short HEAD)
+          echo "SEQR_VERSION=$seqr_version-$commit_id" >> $GITHUB_ENV
+      - uses: actions/checkout@v3
+        with:
+          repository: broadinstitute/seqr-helm
+          ref: main
+      - name: Update appVersion in seqr Chart file
+        uses: mikefarah/yq@v4.22.1
+        with:
+          cmd: >
+            yq -i '.appVersion = "${{ env.SEQR_VERSION }}"' charts/seqr/Chart.yaml
+      - name: Create Pull Request
```

Comment:
Should we push to the main branch directly rather than creating a PR so that it is fully automated?

Reply:
Yes, I think pushing to main (the default branch is "main" rather than "master" on this repo) is probably the right thing to do. If we had a fully automated rollout to prod, then we'd probably want a PR, but I think the plan is to have production pin to a specific commit of the helm chart, so it should be fine to push right to main for this.